aidies-project/db/models/user.js

```diff
@@ -43,6 +47,9 @@ module.exports = (sequelize, DataTypes) => {
     facebook: {
       type: DataTypes.STRING,
     },
+    tokenId: {
+      type: DataTypes.STRING,
+    },
```

Comment:
We can actually remove the `tokenId` because our cookie will take care of giving us the JWT token in the backend!